lib/EnsEMBL/REST/Controller/Info.pm

```diff
@@ -331,6 +331,131 @@ sub consequence_types_GET {
   return;
 }
 
+### The following methods were ported from EG REST...
+
+sub ensgen_version : Chained('/') PathPart('info/eg_version') :
+  ActionClass('REST') : Args(0) { }
+
+sub ensgen_version_GET {
+  my ( $self, $c ) = @_;
+  $c->log()->info("Retrieving EG version from registry");
+  # lazy load the registry
+  $c->model('Registry')->_registry();
```

Comment:
It would be better for the lazy loading trigger to occur in the Registry model rather than here... I presume this is to avoid EG database scans blocking server start.

Reply:
This has gone now